### PR TITLE
UF-5417: Implemented logic for batch import of Stralfors invoices

### PR DESCRIPTION
**Pull request by josc04**

### CODE REVIEW

File: src/main/java/se/sundsvall/seabloader/batchimport/service/BatchImportService.java [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+	public BatchImportService() {
+		objectMapper = new XmlMapper();
+		objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+		objectMapper.configure(SerializationFeature.INDENT_OUTPUT, true);
+	}
+
+	public void execute(String pathToScan) {
+		RequestId.init();
+
+		final var directory = new File(pathToScan);
```
[the remark this answers is not in the document]
Comment (josc04) [reply src/main/java/se/sundsvall/seabloader/batchimport/service/BatchImportService.java:65]: @sonatype-lift ignore

File: src/main/java/se/sundsvall/seabloader/batchimport/service/BatchImportService.java [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+		// Remove pdf from file items in batch and clear entity manager to use minimum memory footprint
+		stralforsFiles.stream().forEach(StralforsFile::removePdf);
+		entityManager.clear();
+	}
+
+	private StralforsFile addPdfData(String directory, StralforsFile stralforsFile) {
+		try {
+			// Read and add pdf as base64 to file object
+			final var inFileBytes = Files.readAllBytes(Paths.get(directory + File.separator + stralforsFile.getName()));
```
[the remark this answers is not in the document]
Comment (josc04) [reply src/main/java/se/sundsvall/seabloader/batchimport/service/BatchImportService.java:117]: @sonatype-lift ignore

File: src/main/java/se/sundsvall/seabloader/batchimport/util/NotificationService.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import generated.se.sundsvall.messaging.EmailRequest;
+import generated.se.sundsvall.messaging.EmailSender;
+import se.sundsvall.seabloader.integration.messaging.MessagingClient;
+import se.sundsvall.seabloader.scheduler.notifier.NotifierSchedulerService;
+
+@Service
+public class NotificationService { // TODO: Remove after completion of Stralfors invoices import
```
Comment (foobar-1337) [inline src/main/java/se/sundsvall/seabloader/batchimport/util/NotificationService.java:19]: Move to service-package? 
(this is a service, not a util?)

File: src/main/java/se/sundsvall/seabloader/batchimport/util/NotificationService.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+	private static final Logger LOGGER = LoggerFactory.getLogger(NotifierSchedulerService.class);
```
Comment (foobar-1337) [inline src/main/java/se/sundsvall/seabloader/batchimport/util/NotificationService.java:21]: Change logger name to NotificationService ?

**Comment by josc04** [comment]
Acted on PR comments

**Review by foobar-1337 — APPROVED** [review]

**Pull request merged**